src/deroff.rs

```diff
@@ -538,12 +538,311 @@ impl Deroffer {
         unimplemented!()
     }
 
+    fn quoted_arg(&self) -> bool {
+        unimplemented!()
+    }
+
+    fn request_or_macro(&mut self) -> bool {
```

Comment:
Can the python code for this be deleted?

Reply:
`quoted_arg` isn't implemented as of right now, but I removed the source for `request_or_macro`